src/kata6.joinNames.js

```diff
@@ -1,3 +1,10 @@
-const joinNames = namesObj => {};
+const joinNames = namesObj => {
+   const names = namesObj.map(nameObject => nameObject.name ) 
+    if(names.length > 1) {
+      const lastName = names.pop()
+     return (names.join(', ')+' & '+lastName)
+     }
+     return names[0];
+   };
```

Comment:
This is a really good solution to the problem. And also perfect on declaring and assigning your variables.